### PR TITLE
Mz post qs

### PR DESCRIPTION
**Pull request by MarZmu**
set up put requests for QA and Review - helpful/report requests

### CODE REVIEW

File: client/src/components/Answer.jsx [hunk 1]
```diff
@@ -1,4 +1,5 @@
 import { useState, React } from 'react';
```
Comment (MarZmu) [inline client/src/components/Answer.jsx:1]: event handler calling put request for answers

File: client/src/components/QAhelp/QArequests.js [hunk 2]
```diff
@@ -0,0 +1,77 @@
+/* eslint-disable camelcase */
+const axios = require('axios');
+
+
```
Comment (MarZmu) [inline client/src/components/QAhelp/QArequests.js:4]: all the QA related post / put request in a single file

File: client/src/components/QModal.jsx [hunk 3]
```diff
@@ -7,12 +7,15 @@ function QModal({ product }) {
 
   function onSubmitClickVal() {
```
Comment (MarZmu) [inline client/src/components/QModal.jsx:8]: question post request event handler

File: client/src/components/Question.jsx [hunk 4]
```diff
@@ -2,28 +2,38 @@ import { React, useState} from 'react';
 import _ from 'underscore';
 import { useSelector } from 'react-redux';
```
Comment (MarZmu) [inline client/src/components/Question.jsx:3]: put request for Questions event handler

File: server/controllers.js [hunk 5]
```diff
@@ -25,3 +30,29 @@ module.exports.get = (endpoint, config = {}) => {
       console.error(new Error(`failed to make get request to API using endpoint ${endpoint}`));
     });
```
Comment (MarZmu) [inline server/controllers.js:31]: put request controller for QA and review

**Review by CotyJ — APPROVED** [review]
Seems fine.
w/Some excess padding/comments that can be refactored out later.

**Pull request merged**